prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -245,8 +245,8 @@ enum fi_ibv_rdm_tagged_buffer_status {
 };
 
 struct fi_ibv_rdm_tagged_buffer_service_data {
-	enum fi_ibv_rdm_tagged_buffer_status status;
-	int seq_number;
+	volatile enum fi_ibv_rdm_tagged_buffer_status status;
+	volatile int seq_number;
```

Comment:
The use of volatile is almost always wrong unless you're dealing with a memory buffer that is shared with hardware.